classes/05class/cicd/README.md

```diff
@@ -0,0 +1,130 @@
+# Continuous Integration / Continuous Deployment
+
+I need to add/delete changes to my running application as automated as possible.
+ 
+***Contents***
+- [Pipeline]()
+- [Continuous Integration]()
```

Comment:
Add Continuous Delivery?

Reply:
tbh as our content is entry-level, I decided to abstract delivery into deployment as they overlap quite a bit.

Reply:
and I think you review an old version that the summary was incomplete haha